src/apps/schedule/crud/events.py

```diff
@@ -204,6 +296,29 @@ async def count_by_applet(
 
         return activity_event_counts
 
+    async def count_by_activity(self, activity_id: int) -> int:
+        """Return event count."""
+
+        query: Query = select(
+            func.count(*[ActivityEventsSchema.event_id]).label("count"),
```

Comment:
`func.count(ActivityEventsSchema.event_id).label('count')`
you can use it in this way